tests/test_decorators.py

```diff
@@ -52,90 +50,84 @@ def test_auth_accepted(self, client, default_guard):
         a valid jwt token, setting the `current_user()`.
         """
 
-        # Token is not in header
+        # Token is not in header or cookie
         response = client.get(
-            '/kinda_protected',
+            "/kinda_protected",
             headers={},
         )
         assert response.status_code == 200
-        assert (
-            "success"
-            in response.json['message']
-        )
-        assert response.json['user'] is None
+        assert "success" in response.json["message"]
+        assert response.json["user"] is None
 
         # Token is present and valid
-        moment = pendulum.parse('2017-05-24 10:38:45')
+        moment = pendulum.parse("2017-05-24 10:38:45")
         with freezegun.freeze_time(moment):
             response = client.get(
-                '/kinda_protected',
+                "/kinda_protected",
                 headers=default_guard.pack_header_for_user(self.the_dude),
             )
             assert response.status_code == 200
-            assert (
-                "success"
-                in response.json['message']
-            )
-            assert response.json['user'] == self.the_dude.username
+            assert "success" in response.json["message"]
+            assert response.json["user"] == self.the_dude.username
 
-    def test_auth_required(self, client, default_guard):
+    def test_auth_required(self, client, default_guard, use_cookie):
         """
         This test verifies that the @auth_required decorator can be used
         to ensure that any access to a protected endpoint must have a properly
         structured auth header including a valid jwt token.  Otherwise,
```

Comment:
Please update docstring to mention that it also covers the case of a token in the cookie.

Reply:
updated.